libtk205/include/libtk205.h

```diff
@@ -0,0 +1,38 @@
+#ifndef LIBTK205_H_
+#define LIBTK205_H_
+
+#include <vector>
+#include <memory>
+#include "ASHRAE205.h"
```

Comment:
This file should be generated from a template to automatically include the header file corresponding to each schema file.